jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/traveler/TimController.java

```diff
@@ -311,16 +311,19 @@ public TimController(OdeProperties odeProperties) {
             ieee.setContent(ieeeContent );
             ieeeDataTag.setIeee1609Dot2Data(ieee);
             
+            byte sendToRsu = travelerInputData.getRsus() != null ? (byte)1:(byte)0;
+            byte[] distroType = new byte[]{(byte) ((byte)(2) | sendToRsu)};
+            
```

Comment:
We need to define named byte constants for the bit-masks (NONE, IP and RSU) and use the the constants instead of hard-coding them.